### PR TITLE
Atomic vehicles for Bright nights mod

### PR DESCRIPTION
**Pull request by John-Candlebury**
This PR adds atomic versions of:
- The Beetle
- The Suv
- Normal cars

Also adds a new gas station with a reinforced vending machine that sells plutonium cells.

### CODE REVIEW

File: data/mods/BrightNights/vehicles/cars.json [hunk 1]
```diff
@@ -0,0 +1,268 @@
+[
+  {
+    "id": "beetle_atomic",
+    "type": "vehicle",
+    "name": "Brightbug",
```
Comment (Coolthulhu) [inline data/mods/BrightNights/vehicles/cars.json]: This sounds too... meta

File: data/mods/BrightNights/vehicle_groups.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
+[
+  {
+    "id": "city_vehicles",
+    "type": "vehicle_group",
+    "vehicles": [
+      [ "car_atomic", 1000 ],
+      [ "car_sports_atomic", 500 ],
```
Comment (Coolthulhu) [inline data/mods/BrightNights/vehicle_groups.json]: Sports cars have the enhanced engines and those would be better off staying rare - that way they'd be more of a reward than "oh, another enhanced engine".

**Comment by Coolthulhu** [comment]
Looks good, I think

**Comment by Coolthulhu** [comment]
I checked them out in game and noticed some problems:
* They are all significantly heavier than any others, which doesn't work well with electric engines. The big vehicles take it fine due to relatively overpowered engine (large electric has 60% more power than v4 in small cars), but the compact can't - look at its acceleration
* I can't get the vehicles to spawn in game without debug. I ran around for long enough to see old atomic sports car, but none of the new ones.
* Ones spawned in debug all had 0 plutonium. Though this seems to be the problem in other vehicles too - I'll look into this

**Comment by John-Candlebury** [comment]
>I can't get the vehicles to spawn in game without debug. I ran around for long enough to see old atomic sports car, but none of the new ones.

Strange, I do get them, they are not common however, about one per medium sized city, will change it. How common do you want them to be? 

See one every street level of common?

>They are all significantly heavier than any others, which doesn't work well with electric engines. The big vehicles take it fine due to relatively overpowered engine (large electric has 60% more power than v4 in small cars), but the compact can't - look at its acceleration

I used the bubble car as a base point for the compact (It has an acceleration of 5mph vs the compacts 7mph) Should I buff both?

**Comment by Coolthulhu** [comment]
> Strange, I do get them, they are not common however, about one per medium sized city. I will change it. How common do you want them to be?

Weird, the numbers on the lists seem huge.
There may be something weird going on.

One per medium city would be fine, though.

> I used the bubble car as a base point for the compact (It has an acceleration of 5mph vs the compacts 7mph) Should I buff both?

Maybe give those vehicles more of those weak electric engines and/or use light parts everywhere? They don't need to be fast, just that they shouldn't be THAT slow.
The "medium" electric engine is tiny and weak, which makes it useless even despite its huge efficiency.

**Comment by John-Candlebury** [comment]
Alright, now the atomic compacts should be comparable to their gasoline counterparts

**Pull request merged**